### PR TITLE
Add chaincode-external for asset-transfer-basic sample.

### PR DESCRIPTION
**Pull request by lehors**
Improve on the initial PR to expand on the README with detailed
information on how to run the external chaincode in a container
on the test network and use the node application against it.

Signed-off-by: Arnaud J Le Hors <lehors@us.ibm.com>

### CODE REVIEW

File: asset-transfer-basic/chaincode-external/README.md [hunk 1]
```diff
@@ -47,37 +45,106 @@ tar cfz code.tar.gz connection.json
...
-## Starting the Asset-Transfer-Basic external service
+This will start the container and start the external chaincode service within it.
+
+## Finish installing the Asset-Transfer-Basic external chaincode
+
+Finishing the installation of the chaincode on the test network can be done from the terminal you started the network from with the following commands (make sure the package-id is set to the right one):
+
+```
+setGlobals 2
+../bin/peer lifecycle chaincode approveformyorg -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --tls --cafile $PWD/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem --channelID mychannel --name basic --version 1.0 --package-id basic_1.0:0262396ccaffaa2174bc09f750f742319c4f14d60b16334d2c8921b6842c090c --sequence 1
```
Comment (DereckLuo) [inline asset-transfer-basic/chaincode-external/README.md:123]: Maybe use a variable to hold `package-id` will be a bit cleaner, it is used in several places, and keep copy and pasting it is a bit tedious.

File: asset-transfer-basic/chaincode-external/sampleBuilder/bin/build [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 echo "!! DL build script invoked"
```
[the remark this answers is not in the document]
Comment (lehors) [reply asset-transfer-basic/chaincode-external/sampleBuilder/bin/build]: Done

File: asset-transfer-basic/chaincode-external/sampleBuilder/bin/detect [hunk 3]
```diff
@@ -1,13 +1,14 @@
-#!/bin/bash
+#!/bin/sh
 
 echo "!! DL detect script invoked"
```
[the remark this answers is not in the document]
Comment (lehors) [reply asset-transfer-basic/chaincode-external/sampleBuilder/bin/detect]: Done

File: asset-transfer-basic/chaincode-external/sampleBuilder/bin/release [hunk 4]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 echo "!! DL release script invoked"
```
[the remark this answers is not in the document]
Comment (lehors) [reply asset-transfer-basic/chaincode-external/sampleBuilder/bin/release]: Done

File: asset-transfer-basic/chaincode-external/README.md (hunk 1 above)
Comment (lehors) [reply asset-transfer-basic/chaincode-external/README.md]: Good idea. I actually did just that eventually in my environment!
Done.

**Comment by lehors** [comment]
> Just couple nits and suggestions.
> Generally LGTM and I'm able to get things up and running very smoothly.

Thanks for reviewing it and trying it out.

**Pull request merged**